controllers/authController.js

```diff
@@ -35,6 +35,7 @@ const githubAuth = (req, res, next) => {
 
       // respond with a cookie
       res.cookie(config.get('userToken.cookieName'), token, {
+        domain: 'realdevsquad.com',
```

Comment:
@ankushdharkar , the domain name won't be the same everytime, e.g, in stage environments or in local.
Can you make this configurable via config files?